tt_metal/hw/inc/firmware_common.h

```diff
@@ -61,6 +67,20 @@ inline void firmware_kernel_common_init(void *init_local_l1_base) {
         (**fptr)();
     }
 }
+
+void noc_bank_table_init(uint64_t mem_bank_to_noc_addr)
+{
+    int32_t num_dram_to_noc_words = (NUM_NOCS * NUM_DRAM_BANKS) << 1;
```

Comment:
can you use sizeof(array)?
this size calculation looks wrong - is size in bytes, right?